core/runtime/impl/wasm_executor.cpp

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "runtime/impl/wasm_executor.hpp"
+
+#include <binaryen/wasm-binary.h>
+#include "runtime/impl/runtime_external_interface.hpp"
+
+namespace kagome::runtime {
+
+  WasmExecutor::WasmExecutor(std::shared_ptr<extensions::Extension> extension)
+      : extension_(std::move(extension)) {}
+
+  wasm::Literal WasmExecutor::call(const std::vector<uint8_t> &code,
+                                   wasm::Name method_name,
+                                   const wasm::LiteralList &args) {
+    wasm::Module module{};
```

Comment:
is it necessary to allocate Module for every `call`?

Reply:
Good point. I actually looked at how it is done in [hera](https://github.com/ewasm/hera/blob/master/src/binaryen.cpp#L475). If we want to optimize this in future we will use method call in wasm which is also implemented now. 
Currently, not clear if we need to instantiate new module on every runtime api call